jaeger-core/src/test/java/io/jaegertracing/internal/JaegerSpanTest.java

```diff
@@ -210,14 +209,11 @@ public void testWithoutTimestampsInaccurateClock() {
 
   @Test
   public void testSpanToString() {
-    JaegerSpan jaegerSpan = tracer.buildSpan("test-operation").start();
-    JaegerSpanContext expectedContext =  jaegerSpan.context();
-    JaegerSpanContext actualContext = TextMapCodec.contextFromString(expectedContext.contextAsString());
-
-    assertEquals(expectedContext.getTraceId(), actualContext.getTraceId());
-    assertEquals(expectedContext.getSpanId(), actualContext.getSpanId());
-    assertEquals(expectedContext.getParentId(), actualContext.getParentId());
-    assertEquals(expectedContext.getFlags(), actualContext.getFlags());
+    String operation = "test-operation";
+    JaegerSpan span = tracer.buildSpan(operation).start();
+    String expectedString = span.context().toString() + " - " + operation;
```

Comment:
prod == prod.  Please test for exact string.

Reply:
You mean I should use a string literal instead of `expectedString`? Not sure what the suggestion is.

Reply:
Right. Will do.